ui/src/app/applications/components/utils.tsx

```diff
@@ -534,6 +539,34 @@ export function renderResourceMenu(
     );
 }
 
+export function renderResourceActionMenu(resource: ResourceTreeNode, application: appModels.Application, tree: appModels.ApplicationTree, appContext: AppContext): React.ReactNode {
+    let menuItems: Observable<ActionMenuItem[]>;
+
+    menuItems = getResourceActionsMenuItems(resource, application, appContext);
```

Comment:
Could this be compressed with the line above so that it's
 
```suggestion
   const menuItems = getResourceActionsMenuItems(resource, application, appContext);
```